Makefile.am

```diff
@@ -80,14 +80,15 @@ RPM_BUILD_DIR = $(BUILD_RESULT_DIR)/01-rpm-build
 TEST_INST_BUILD_DIR = $(BUILD_RESULT_DIR)/02-test-install
 
 # build/run tweaks for all containers, such as --cap-add
+CONTAINER_ENGINE ?= podman
 CONTAINER_BUILD_ARGS ?=
 CONTAINER_TEST_ARGS ?=
 
 # ci-tasks container
 CI_TASKS_DOCKERFILE ?= $(srcdir)/dockerfile/ci-tasks
 CI_TASKS_NAME ?= anaconda/ci-tasks
 CI_TASKS_TAG ?= $(GIT_BRANCH)
-CI_TASKS_TEST_ARGS ?= --rm -v .:/anaconda:Z
+CI_TASKS_TEST_ARGS ?= --rm -v $(CURDIR):/anaconda:Z
```

Comment:
I did not know about this variable. Pretty nice!

Don't we want to use this instead of ``$(srcdir)`` above too?

Reply:
Good point, thanks! I cleaned this up by having srcdir default to `$(CURDIR)` instead of `.` and using `$(srcdir)` here.